code/game/machinery/doors/airlock.dm

```diff
@@ -1216,9 +1216,14 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 			else
 				close(1)
 		return
-	else if(!ispowertool(I))
+	else if(!ispowertool(I) && !istype(I, /obj/item/melee/mantis_blade))
 		to_chat(user, "<span class='warning'>The airlock's motors resist your efforts to force it!</span>")
 		return
+	if(istype(I, /obj/item/melee/mantis_blade))
+		var/obj/item/melee/mantis_blade/secondblade = user.get_inactive_hand()
+		if(!istype(secondblade, /obj/item/melee/mantis_blade))
+			to_chat(user, "<span class='warning'>You need a second [I] to pry open doors!</span>")
+			return
```

Comment:
Hmm, this feels a bit snowflakey. I think it'd be best to make a quick door-forcing a component with some signals, if you could, and have it applied to items that can force doors.
Feel free to reach out on discord if you have any questions.